test-data/unit/check-literal.test

```diff
@@ -3149,3 +3149,40 @@ err_code = -TWO
 if bool():
     err_code = -THREE
 [builtins fixtures/float.pyi]
+
+[case testNestedClassOnAliasAsType]
+from typing_extensions import Literal
```

Comment:
Actually one more thing, sorry for not noticing this before. This import is unused, and the test case can be moved to `check-type-aliases.test`.

Reply:
Wow, I didn't notice that unused Literal import either, great catch!